README.md

```diff
@@ -42,17 +42,22 @@ async ()=> {
 
 #### Transpilation / Polyfill
 
-There is no way of transpilating the actual effect of this proposal. But instead we can write a polyfill that keeps API consistent while it's using the main thread for parsing and stringifying tasks.
+There is no way of transpilating the actual effect of this proposal. But instead we can write a polyfill that keeps API consistent while it's using the main thread for parsing and stringifying tasks. We are using promises to perform the sync operation in the next processor tick.
 
 
 ```js
 JSON.stringifyAsync = JSON.stringifyAsync || function (object) {
-  return new Promise(resolve => resolve(JSON.stringify(object)));
+  return new Promise(resolve => {
```

Comment:
Actually we completely forgot about other arguments in the polyfill.

So it should be:

```js
JSON.parseAsync = JSON.stringifyAsync || function () {
  return Promise.resolve.apply(null, arguments).then(JSON.stringify)
}
```
will that pass all the arguments to `stringify`?